### PR TITLE
Coretest fix

### PR DESCRIPTION
**Pull request by ghost**
Fixed tests failing.

### CODE REVIEW

**Comment by ghost** [comment]
@amotenko @OriHoch The Travis build is not failing, but erring out and stopping due to Yarn timing out on fetching one of the packages.
This work locally with the same env + and versions of node, npm and yarn AFAICT

Most likely a temp issue with Travis/Yarn registry.
Will try to retrig the build later.

`error An unexpected error occurred: "https://registry.yarnpkg.com/async/-/async-2.6.0.tgz: ETIMEDOUT".`

**Comment by OriHoch** [comment]
@elad-lachmi  the original failure was due to tests - https://travis-ci.org/Midburn/spark/jobs/330892147
I don't know what is the failure you are referring to..

File: package-lock.json.old [hunk 1]
```diff
@@ -0,0 +1,9432 @@
+{
```
Comment (OriHoch) [inline package-lock.json.old:1]: I think this file can be removed

File: models/constants.js [hunk 2]
```diff
@@ -197,6 +197,12 @@ module.exports = {
...
     CAMP_MEMBER_STATUS: ['approved', 'pending', 'pending_mgr', 'rejected', 'approved_mgr', 'supplier', 'deleted'],
 
-    EVENT_GATE_STATUS: get_enum(gate_status)
+    EVENT_GATE_STATUS: get_enum(gate_status),
+
+    // This is a list of URLs the login process is allowed to redirect to.
+    // This is to make sure users are not sent spark links via e.g. email by a malicious 3rd party
+    // and are redirected to the senders desired location, e.g. can be used for phishing.
+    // If we redirect to a new URL from login, we will need to add it here.
+    LOGIN_REDIRECT_URL_WHITELIST: ['/', '/admin'],
```
Comment (OriHoch) [inline models/constants.js:206]: I think this list should contain a whitelist of domain names (paths are easy to fake)
on production this list would be `["volunteers.spark.midburn.org","spark.midburn.org","dreams.midburn.org"]`
you can have it coming from environment variable and set for each environment

**Review by OriHoch — APPROVED** [review]
wrote a couple of minor comments, but good enough to merge

File: models/constants.js (hunk 2 above)
Comment (ghost) [reply models/constants.js:206]: This is just a redirect from login, i.e. the r QS parameter.
If dreams, volunteers, etc. use the same login, then sure, we could do that.

**Pull request merged**